R/run_app.R

```diff
@@ -55,10 +55,14 @@ run_app <- function(
       credentials_db_name <- "credentials_dev.sqlite"
       login_creds <- list(user_id = "admin",
                           user_pwd = "cxk1QEMYSpYcrNB")
-    } else if (pre_auth_user == "nonadmin") {
+    } else if (pre_auth_user == "lead") {
       credentials_db_name <- "credentials_dev.sqlite"
-      login_creds <- list(user_id = "nonadmin",
+      login_creds <- list(user_id = "lead",
                           user_pwd = "Bt0dHK383lLP1NM")
+    } else if (pre_auth_user == "reviewer") {
+      credentials_db_name <- "credentials_dev.sqlite"
```

Comment:
Perhaps we could remove the redundancy of assigning `credentials_db_name` in each if-else-then at the top since you are already checking if it's not `NULL`?

```r
credentials_db_name <- "credentials_dev.sqlite"
```

Reply:
I have removed the redundancy.